ts/Grid/Core/Grid.ts

```diff
@@ -340,15 +339,15 @@ class Grid {
     }
 
     /**
-     * Initializes the container of the data grid.
+     * Initializes the container of the Grid.
      *
      * @param renderTo
-     * The render target (html element or id) of the data grid.
+     * The render target (html element or id) of the Grid.
      *
      */
     private initContainers(renderTo: string|HTMLElement): void {
         const container = (typeof renderTo === 'string') ?
-            win.document.getElementById(renderTo) : renderTo;
+            document.getElementById(renderTo) : renderTo;
```

Comment:
Don't we need to use window reference here? Will it work in frameworks like react etc?